### DISCUSSION TITLE
NEW UPDATE WORKAROUND?

### DISCUSSION THREAD

**Discussion by MODSAREFUNwhoknew** [praydog/REFramework#378] [General Questions]
Does anyone know how to not update the game and still play it on Steam, does it support rollback playability somehow?

**Comment by MODSAREFUNwhoknew** [comment]
I'm recommending backing up the game files until Praydog has an answer for the new update.  Hope you all are having a great day.

**Reply by darkmetalman** [reply]
I did not backup RE2 and I cant play it anymore on VR =( 
 I did backup RE3 and I can play it by going offline, lets hope Praydog can find a workaround this new patch.

**Comment by IronbugVR** [comment]
You got to set the games to update when I start them, then only play them offline. Only work around I have found.
